js/views/playingScreen.js

```diff
@@ -175,10 +177,10 @@ var PlayingScreen = React.createClass({
     // for mobile, touch is handled in handleTouchEnd
     if (this.props.controller.videoVr) {
       this.setState({
-        isVRMouseDown: false,
+        isVrMouseDown: false,
       });
       
-      if (this.props.controller.checkVrDirection) {
+      if (_.isFunction(this.props.controller.checkVrDirection)) {
```

Comment:
Let's use typeof X === "function" instead of using extra underscore library load.

Reply:
\+